.github/workflows/test.yml

```
@@ -50,19 +107,51 @@ jobs:
             eslint: 4
           - node: 6
             eslint: 3
-
-    # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
-        # Checkss out the repository under $GITHUB_WORKSPACE
       - name: Checkout
         uses: actions/checkout@v2
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-node-modules
+        with:
+          # npm cache files are stored in `~/.npm` on Linux/macOS
+          path: ~/.npm
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
       - name: Load Node version ${{ matrix.node }}
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node }}
       - name: Npm Install
-        run: npm install
+        run: npm ci
       - name: Install ESLint version ${{matrix.eslint}}
         run: npm uninstall --no-save eslint-config-airbnb-base && npm install --no-save "eslint@${{matrix.eslint}}"
       - name: Run ESLint
-        run: npm run test:ci
+        run: npm run lint
+  diff-breakUpAriaJSON:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-node-modules
+        with:
+          # npm cache files are stored in `~/.npm` on Linux/macOS
+          path: ~/.npm
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+      - name: Load Node version
+        uses: actions/setup-node@v1
```

v2 of setup-node is the stable release now
```suggestion
        uses: actions/setup-node@v2
```

---

👍